src/library.js

```diff
@@ -467,7 +467,7 @@ LibraryManager.library = {
   },
 
   emscripten_get_heap_size: function() {
-    return HEAP8.length;
+    return HEAPU8.length;
```

Comment:
This saves code bytes, HEAPU8 is referenced in preamble by default to set up the heap, referencing via that avoids needing to set up both HEAP8 and HEAPU8 in small applications.

Reply:
Please add a comment as otherwise it's very possible this will get "optimized" back eventually...

Reply:
I see this was resolved without a change - maybe by mistake? Or do you think a comment isn't worth it?